### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by rossf7**
See https://github.com/giantswarm/metrics-server-app/pull/5#issue-324689281.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
 [![CircleCI](https://circleci.com/gh/giantswarm/kube-state-metrics-app.svg?style=svg)](https://circleci.com/gh/giantswarm/kube-state-metrics-app)
 
 # kube-state-metrics Helm Chart
+
 Helm Chart for kube-state-metrics in Tenant Clusters.
 
-* Installs the [kube-state-metrics agent](https://github.com/kubernetes/kube-state-metrics).
+* Installs the [kube-state-metrics agent].
```
Comment (puja108) [inline README.md]: why did you remove the ink here?

File: README.md [hunk 2]
```diff
@@ -21,4 +28,14 @@ Provide a custom `values.yaml`:
...
 
-Deployment to Tenant Clusters is handled by [app-operator](https://github.com/giantswarm/app-operator).
+## Release Process
+
+* Ensure CHANGELOG.md is up to date.
+* Create a new GitHub release which will push the tarball to the [default-catalog].
+* Update [cluster-operator] with the new version.
+
+[app-operator]: https://github.com/giantswarm/app-operator
+[cluster-operator]: https://github.com/giantswarm/app-operator
```
Comment (puja108) [inline README.md]:
```suggestion
[cluster-operator]: https://github.com/giantswarm/cluster-operator
```

**Comment by rossf7** [comment]
Link is fixed. PTAL

**Comment by rossf7** [comment]
Same as other apps repos. Going with this.
